src/pytest_servers/s3.py

```diff
@@ -36,3 +44,44 @@ def close(self):
 
     def __exit__(self, *exc_args):
         self.close()
+
+
+@pytest.fixture(scope="session")
+def s3_fake_creds_file(monkeypatch_session):
+    # https://github.com/spulec/moto#other-caveats
+    import pathlib
+
+    aws_dir = pathlib.Path("~").expanduser() / ".aws"
+    aws_dir.mkdir(exist_ok=True)
+
+    aws_creds = aws_dir / "credentials"
+    initially_exists = aws_creds.exists()
+
+    if not initially_exists:
+        aws_creds.touch()
+
+    try:
+        with monkeypatch_session.context() as m:
+            try:
+                m.delenv("AWS_PROFILE")
+            except KeyError:
+                pass
+            m.setenv("AWS_ACCESS_KEY_ID", "pytest-servers")
+            m.setenv("AWS_SECRET_ACCESS_KEY", "pytest-servers")
+            m.setenv("AWS_SECURITY_TOKEN", "pytest-servers")
+            m.setenv("AWS_SESSION_TOKEN", "pytest-servers")
+            m.setenv("AWS_DEFAULT_REGION", "eu-south-1")
```

Comment:
Seems different from `aws_region` fixture

Reply:
Got rid of this, we'll add it back it it creates any trouble